requirements.txt

```diff
@@ -5,3 +5,4 @@ django-registration>=3.0
 pytz>=2018.9
 ordereddict>=1.1
 pyyaml>=4.2b1
+numpy<2
```

Comment:
Why do we need to add numpy ?

Reply:
it's because numpy>=2 is breaking some dependencies (see https://stackoverflow.com/questions/78634235/numpy-dtype-size-changed-may-indicate-binary-incompatibility-expected-96-from )